essentials/function.mdx

```diff
@@ -325,6 +325,132 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
 ```
 It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
 ```
+### Example 4: `get_email_resume`
+For this example we will use gmail as an email service
+- Objective: Create a resume from your e-mails
+- Parameter: `value` (desired quantity of e-mails), `login` (your e-mail), `password` (your e-mail password)
+- Usage Example: User provides a location and receive local weather information
+
+### Request
+Firstly, you need to create a password for less secure apps by following the link: https://support.google.com/a/answer/6260879?hl=en.
+With the password created, we can make the first request to extract the email, password and number of emails that will be summarized.
+```python
+api_request_json = {
+  "model": "llama-70b-chat",
+  "messages": [
+    {"role": "user", "content": "Make a resume from my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},
```

Comment:
This is a big security problem.... your email and password are now public to the world! I imagine you already expired your password... but change your email so you don't get too much SPAM messages

Reply:
https://stackoverflow.com/questions/2110595/redaction-in-git